Firebase/android/app/src/main/java/com/novoda/bonfire/view/BubblyDrawable.java

```diff
@@ -75,4 +58,18 @@ private void drawBubble(Canvas canvas, int xPosition, int yPosition, int diamete
         canvas.drawRoundRect(rect, radius, radius, paint);
     }
 
+    @Override
+    public void setAlpha(int alpha) {
+        paint.setAlpha(alpha);
+    }
+
+    @Override
+    public void setColorFilter(ColorFilter colorFilter) {
+        paint.setColorFilter(colorFilter);
+    }
+
+    @Override
+    public int getOpacity() {
+        return PixelFormat.OPAQUE;
```

Comment:
Should be `TRANSLUCENT` unless _every_ pixel within the drawable's bounds is 100% opaque! ⚠️